web/views/styleguide_view.ex

```diff
@@ -1,3 +1,100 @@
 defmodule App.StyleGuideView do
+  @moduledoc false
   use App.Web, :view
+
+  @doc """
+  ## Render both the component and its code
+  iex> App.StyleGuideView.render_whole_component("./test/support/example_components")
```

Comment:
If you write `doctest App.StyleGuideView, import: true` in the test, then this can be shortened to:
```
iex> render_whole_component("./test/support/example_components")
```

Which is a bit nicer (only just found this [here](http://stackoverflow.com/questions/43242418/elixir-how-can-you-use-an-alias-in-doctest/43246122#43246122)